tails_files/install.sh

```diff
@@ -20,15 +46,48 @@ apt-get update
 apt-get install -y build-essential
 gcc -o $SCRIPT_BIN securedrop_init.c
```

Comment:
The binary built by gcc is output to the persistent install dir:
SCRIPT_BIN=$INSTALL_DIR/securedrop_init